scenarios/tracking/01_training_introduction.ipynb

```diff
@@ -20,7 +20,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Is this safe and load needed, or here mainly as an example? If example, then could comment out the code and mention that this is an example to save/load a model. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/553/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Mainly as an example - but also because if you just run in straight through, we only train for 1 epoch so the subsequent <code>predict</code>  runs would output basically no tracks. This is why I show how to load weights from a model Karine trained  for 30 epochs before calling <code>predict.</code> I can wrap the <code>save</code> method in <code>if SAVE_MODEL = True</code>, but what do you think would be the best way to handle the <code>load</code> examples? </p>

Reply:
Sounds like we can leave this as is for now since it helps us speed up model development (due to us not wanting to wait 30 epochs every time we run the notebook). Longer term I would suggest to just comment out the two lines and write a comment that this is how one would load/save the model.